### PR TITLE
Sch 193318 oascard

### PR DESCRIPTION
**Pull request by shewood**
## [ADO-193318](https://dev.azure.com/VP-BD/DECD/_workitems/edit/193318)

[AB#193318](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/193318)

### Description of proposed changes:
Added a new card for OAS testing for Curam that can easily be modified or removed if no longer needed. 

### Additional Notes
Still need to add the link for the most requested item.

### CODE REVIEW

**Comment by shewood** [comment]
closing.

File: cypress/e2e/dashboard.cy.js [hunk 1]
```diff
@@ -35,13 +35,23 @@ describe('Validate dashboard page', () => {
...
-    cy.get('[data-cy="cardtitle"]').should('be.visible').and('have.length', 7)
+  it('Validate 5 Cards (EI,CPP,OAS,SIN,CAL) +2 cards for (CPCD and OAS) and Card titles are Visible', () => {
+    const cardTitles = [
+      'Canadian Dental Care Plan',
+      'Employment Insurance',
+      'Canada Pension Plan',
+      'Old Age Security',
+      'Social Insurance Number',
+      'Canada Apprentice Loan',
+      'Old Age Security',
```
Comment (krischarbonneau) [inline cypress/e2e/dashboard.cy.js:46]: I think we could potentially call this something else like `OAS on Curam` instead to differentiate the two different cards but this is fine for now

**Review by krischarbonneau — APPROVED** [review]
LGTM

File: cypress/e2e/dashboard.cy.js (hunk 1 above)
Comment (shewood) [reply cypress/e2e/dashboard.cy.js:46]: Good Catch, I updated the tests for this issue.

**Pull request merged**